grab.js

```diff
@@ -480,6 +480,25 @@ var MoveGrab = class MoveGrab {
         // ensure window is properly activated after layout/ensureViewport tweens
         Mainloop.timeout_add(0, () => {
             Main.activateWindow(metaWindow);
+
+            // if floating or scratch, then exit (no need to click-out)
+            if (this.initialSpace.isFloating(metaWindow) ||
+                Scratch.isScratchWindow(metaWindow)) {
+                return;
+            }
+            /**
+             * Gnome 44 removed the ability to manually end_grab_op.
+             * Previously we would end the grab_op before doing
+             * PaperWM grabs.  In 44, we can't do this so the grab op
+             * may still be in progress, which is okay, but won't be ended
+             * until we "click out".  We do this here if needed.
+             */
+            if (!global.display.end_grab_op) {
+                getVirtualPointer().notify_button(Clutter.get_current_event_time(),
+                    Clutter.BUTTON_PRIMARY, Clutter.ButtonState.PRESSED);
+                getVirtualPointer().notify_button(Clutter.get_current_event_time(),
+                    Clutter.BUTTON_PRIMARY, Clutter.ButtonState.RELEASED);
```

Comment:
Where is this actually clicking? Just the current cursor position? Is there any danger of this clicking on e.g. a button in the UI if we drag the window with Super+Drag?

Like for the scratch window this was causing an issue when dragging from the titlebar. But could this also happen elsewhere?

Reply:
Good question - I believe it's at the current mouse cursor position.  It might be safer to move the cursor to an absolute position.

The original issue can be seen by comments the click mechanism out, and then dragging a window by the titlebar - after dragging is complete, you need to "click again" to end the default gnome grab action (it's interfered with by paperwm's grab/drag system).

Reply:
I'll do some testing here to confirm click position.

Reply:
Hey @Lythenas - good pickup, yes, was clicking somewhere (likely 0,0? or maybe whatever default was... indeterminate?) - in latest commit we now move ensure cursor is at the current cursor pos.